grid-view-common.ts

```diff
@@ -141,4 +142,26 @@ export const colWidthProperty = new CoercibleProperty<GridViewBase, PercentLengt
         target.refresh();
     }
 });
-colWidthProperty.register(GridViewBase);
\ No newline at end of file
+colWidthProperty.register(GridViewBase);
+
+export type GridViewOrientation = "vertical" | "horizontal";
+const defaultOrientation: GridViewOrientation = "vertical";
+export const orientationProperty = new CoercibleProperty<GridViewBase, GridViewOrientation>({
```

Comment:
Lets make the property similar to the orientation property of the core `ScrollView` widget: https://github.com/NativeScript/NativeScript/blob/3e38eb4b2f931a8a985037f52d31fdf3d09e456e/tns-core-modules/ui/scroll-view/scroll-view-common.ts#L90 
* No need to be coercible
* Create a converter by importing `makeParser` and `makeValidator` from `ui/core/properties` and use that one as the user might send an invalid value 😄 
* Set the `affectsLayout` of the property to `true`